LinkedList.java

```diff
@@ -97,29 +98,38 @@ public static LinkedList removeFromPosition(LinkedList list,int pos){
             } 
             
 		}
-		return list;
+		
 		}
-		public static LinkedList removeFromHead(LinkedList list){
-			Node temp=list.head;
-			list.head=temp.next;
+		public void removeFromHead(){
+			Node temp=this.head;
+			this.head=temp.next;
 			temp.next=null;
-			return list;
+			
 		}
-		public static LinkedList removeFromLast(LinkedList list){
-			Node last=list.head,prev=null;
+		public void removeFromLast(){
+			Node last=this.head,prev=null;
 			while(last.next!=null){
 				prev=last;
 				last=last.next;
 			}
 			prev.next=null;
-			return list;
+			 
+		}
+	public static int size(){
+		Node temp=front;
+		int count=0;
+		while(temp!=null){
+			count++;
+			temp=temp.next;
 		}
+        return count;
+	}
 					
 					
                   					
-		public static void printList(LinkedList list)
+		public void printList()
```

Comment:
Again, use toString(), not this. Check abhijeet's code if u don't know how to implement this